content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Get Access to Kubernetes Cluster"
+weight = 10
++++
+
+You can either login to created EC2 instance and use the `/etc/kubernetes/admin.conf` or if you want to
+have access to your Master Kubernetes cluster locally, follow these steps:
+
+### Retrieve Terraform State
```

Comment:
I think we should use level two headers (`##`) instead of level three. Also, I'd use some notation to highlight that this is the first, second, third step, for example:

> 1. Retrieve Terraform State
> 2. Prepare SSH Agent with SSH Key
> 3. Use KubeOne to retrieve kubeconfig